vars/getVaultSecret.groovy

```diff
@@ -58,11 +60,11 @@ def readSecret(secret) {
   return props
 }
 
-def readSecretWrapper(body) {
+def readSecretWrapper(role_id, secret_id, body) {
```

Comment:
this change breaks a few steps and pipelines https://github.com/search?q=org%3Aelastic+readSecretWrapper&type=code

Reply:
Hrm. OK. I guess we'll have to go ahead and create a diverging code path for the needed params, then. I'll do that and restore the original method signature.

Reply:
Done.